x-pack/legacy/plugins/index_management/public/app/components/mappings_editor/components/document_fields/field_parameters/orientation_parameter.tsx

```diff
@@ -22,7 +22,7 @@ export const OrientationParameter = ({ defaultToggleValue }: { defaultToggleValu
       'xpack.idxMgmt.mappingsEditor.geoShapeType.orientationFieldDescription',
       {
         defaultMessage:
-          'Define how to interpret vertex order for polygons / multipolygons. This parameter defines one of two coordinate system rules (Right-hand or Left-hand).',
+          'Define how to interpret vertex order for polygons and multipolygons. This parameter defines one of two coordinate system rule, counterclockwise or clockwise.',
```

Comment:
Maybe we can simplify this wording:

> Interpret the vertex order for polygons and multipolygons as either counterclockwise (default) or clockwise.

Reply:
This order sounds better to me:

clockwise or counterclockwise (default).